source/backend/api/Services/LeaseService.cs

```diff
@@ -84,7 +111,7 @@ private PimsLease AssociatePropertyLeases(PimsLease lease, bool userOverride = f
         {
             MatchProperties(lease);
 
-            lease.PimsPropertyLeases.ForEach(propertyLease =>
+            foreach (var propertyLease in lease.PimsPropertyLeases)
```

Comment:
What is the benefit of making this change?

Reply:
The ForEach was conflicting with the LINQ namespace and the extensions we have